homeassistant/components/somfy/.translations/en.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
These are the wrong translations files. You need to define `strings.json`. These files are automatically generated.